java/ql/src/semmle/code/java/dataflow/internal/ContainerFlow.qll

```diff
@@ -88,6 +91,290 @@ class ContainerType extends RefType {
   }
 }
 
+private class ContainerFlowSummaries extends SummaryModelCsv {
+  override predicate row(string row) {
+    row =
+      [
+        "java.util;Map<>$Entry;true;getValue;;;MapValue of Argument[-1];ReturnValue;value",
+        "java.util;Map<>$Entry;true;setValue;;;MapValue of Argument[-1];ReturnValue;value",
+        "java.util;Map<>$Entry;true;setValue;;;Argument[0];MapValue of Argument[-1];value",
+        "java.lang;Iterable;true;iterator;();;Element of Argument[-1];Element of ReturnValue;value",
+        "java.lang;Iterable;true;spliterator;();;Element of Argument[-1];Element of ReturnValue;value",
+        "java.util;Iterator;true;next;;;Element of Argument[-1];ReturnValue;value",
+        "java.util;ListIterator;true;previous;;;Element of Argument[-1];ReturnValue;value",
+        "java.util;ListIterator;true;add;(Object);;Argument[0];Element of Argument[-1];value",
+        "java.util;ListIterator;true;set;(Object);;Argument[0];Element of Argument[-1];value",
+        "java.util;Enumeration;true;asIterator;;;Element of Argument[-1];Element of ReturnValue;value",
+        "java.util;Enumeration;true;nextElement;;;Element of Argument[-1];ReturnValue;value",
+        "java.util;Map;true;computeIfAbsent;;;MapValue of Argument[-1];ReturnValue;value",
+        "java.util;Map;true;computeIfAbsent;;;ReturnValue of Argument[1];ReturnValue;value",
+        "java.util;Map;true;computeIfAbsent;;;ReturnValue of Argument[1];MapValue of Argument[-1];value",
+        "java.util;Map;true;entrySet;;;MapValue of Argument[-1];MapValue of Element of ReturnValue;value",
+        "java.util;Map;true;entrySet;;;MapKey of Argument[-1];MapKey of Element of ReturnValue;value",
+        "java.util;Map;true;get;;;MapValue of Argument[-1];ReturnValue;value",
+        "java.util;Map;true;getOrDefault;;;MapValue of Argument[-1];ReturnValue;value",
+        "java.util;Map;true;getOrDefault;;;Argument[1];ReturnValue;value",
+        "java.util;Map;true;put;;;MapValue of Argument[-1];ReturnValue;value",
```

Comment:
Same here I suppose -- the old value is returned, but we're presumably going to wire these steps together and alias the new one